src/main/java/com/homework/springboot/service/impl/WRuntimeException.java

```diff
@@ -0,0 +1,11 @@
+package com.homework.springboot.service.impl;
+
+public class WRuntimeException extends Throwable {
```

Comment:
1. Why did you extend from Throwable? Why not from Exception or RuntimeException. 
Please write your explanation here. 
2. What about naming? It is not so good.